packages/extension-api/src/extension-api.d.ts

```diff
@@ -812,6 +812,38 @@ declare module '@podman-desktop/api' {
     buttons?: readonly QuickInputButton[];
   }
 
+  /**
+   * Options to configure the behavior of the message box UI.
+   */
+  export interface MessageBoxOptions {
+    /**
+     * The title of the message box.
+     */
+    title: string;
+    /**
+     * The primary message.
+     */
+    message: string;
+    /**
+     * Text for buttons.
+     */
+    buttons: string[];
+    /**
+     * The (optional) type, one of 'none' | 'info' | 'error' | 'question' | 'warning'.
+     */
+    type?: string;
+
+    /* Future properties to match more of the Electon message box API:
+    checkboxChecked?: boolean;
+    checkboxLabel?: string;
+    detail?: string;*/
+  }
+
+  export interface MessageBoxReturnValue {
+    response: number;
+    checkboxChecked: boolean | undefined;
+  }
+
```

Comment:
AFAIK we already have all the API in `showInformationMessage` `showWarningMessage` or `showErrorMessage` (and with `MessageOptions`)
so no need to add something new in that file

Reply:
we should not expose more options than the one already defined in the showXXX methods.


Extensions have showInputBox to grab things.

IMHO telemetry should be a custom widget and then not use any of these API

Because we want to add links for the details that we are collecting, etc that will be never be possible.

Reply:
Out of date, API removed with new commit.